configure.ac

```diff
@@ -0,0 +1,8 @@
+AC_INIT
+# Check that we have a C compiler
+AC_PROG_CC
+AC_DEFINE(_XOPEN_SOURCE, 700)
```

Comment:
Does this end up being defined when compiling yarp (in this repo, outside of ruby/ruby)?
I don't see anything that would include the file create by autotools, am I missing it?
It's done via `@DEFS@` maybe?

Reply:
Indeed, that's how it works:
```
log:
gcc -g -O2 -std=c99 -Wall -Werror -Wextra -Wpedantic -Wsign-conversion -fPIC	-fvisibility=hidden -DPACKAGE_NAME=\"\" -DPACKAGE_TARNAME=\"\" -DPACKAGE_VERSION=\"\" -DPACKAGE_STRING=\"\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -D_XOPEN_SOURCE=700 -DHAVE_MMAP=1 -DHAVE_STRNCASECMP=1 -Iinclude  -c -o src/enc/shift_jis.o src/enc/shift_jis.c
in Makefile:
DEFS := -DPACKAGE_NAME=\"\" -DPACKAGE_TARNAME=\"\" -DPACKAGE_VERSION=\"\" -DPACKAGE_STRING=\"\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -D_XOPEN_SOURCE=700 -DHAVE_MMAP=1 -DHAVE_STRNCASECMP=1
```